### ISSUE TITLE
Browser not refreshing on code change

### ISSUE THREAD

**Issue by carlosloureda** [diegohaz/arc#338]
I am very new with babel and react, just felt right to test ARC design and this package. I am missing a feature that comes right out of the box with create-react-app command. This feature is the ability to reload the page when changing any line of code. I don't know it I can tweak this with some kind of package.json configuration or something like that. Thank you!

**Comment by vegeta897** [comment]
Are you running `npm run-script dev` or `npm run-script dev-example`?

**Comment by carlosloureda** [comment]
Nope. I am running `npm run dev` . For what I see in my project it refreshes on some action/reducers code changes but not in pages/components code, seems weird.
